sk/music-optimized/Menu.asm

```diff
@@ -605,19 +605,23 @@ Snd_Menu_Jump05:
 	smpsCall            Snd_Menu_Call01
 	smpsCall            Snd_Menu_Call01
 	smpsCall            Snd_Menu_Call01
-	smpsFMAlterVol      $FD, $D3
+	; The following smpsFMAlterVol calls are horribly-bugged: not only
+	; do they not actually do anything on the PSG channel, but they also
+	; cause note data to be skipped.
+	; The dev obviously intended to use smpsPSGAlterVol commands here.
+	;smpsFMAlterVol      $FD, nMaxPSG1
```

Comment:
Hm. I will have to hear the difference in this case. What happens if you change the calls to smpsPSGAlterVol?

Reply:
Oh, nevermind, I read the commit messages and you already tested it and it desynchs.

So I will change to a different question: what happens if you change these to smpsPSGAlterVol and discard the second parameter? The first parameter seems to be generally paired in a balanced manner, and maybe it will yield a clue as to how the additional data is meant to be used.